townhall/myapi/dao.py

```diff
@@ -251,3 +253,44 @@ def filtered_organization(
             filters["location__icontains"] = filtered_organization_data.location
 
         return Organization.objects.filter(**filters)
+
+class TaskDao:
+
+    @staticmethod
+    def get_all_tasks():
+
+        return Task.objects.all()
+
+    @staticmethod
+    def get_task_by_id(task_id: int):
+
```

Comment:
not sure why theres whitespace after every method definition, it shouldnt be there generally